tests/runner_templates/checkout-test-plots.include.yml

```diff
@@ -3,7 +3,7 @@
       run: date +%F > today.txt
 
     - name: Cache test blocks and plots
-      uses: actions/cache@v2
```

Comment:
@AmineKhaldi the jobs run this [pre-commit.yml](https://github.com/Chia-Network/chia-blockchain/blob/main/.github/workflows/pre-commit.yml), but then they check to see if the workflow files match after running [build-workflows.py](https://github.com/Chia-Network/chia-blockchain/blob/main/tests/build-workflows.py)